<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Sergiys-MacBook-Pro.local" name="Surefire test" tests="2" failures="1" timestamp="20 Mar 2017 22:51:35 GMT" time="3.713" errors="0">
  <testcase name="getPushConfiguration" time="2.124" classname="com.eodessa.test.PushConfigurationTest"/>
  <testcase name="validatePathParameter" time="1.569" classname="com.eodessa.test.PushConfigurationTest">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
	at com.eodessa.test.PushConfigurationTest.validatePathParameter(PushConfigurationTest.java:62)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- validatePathParameter -->
</testsuite> <!-- Surefire test -->
